src/components/Navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Navbar = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+          to="/"
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+          to="/people"
+        >
+          People
+        </NavLink>
```

Comment:
are you sure it's the right link?
<img width="554" alt="image" src="https://user-images.githubusercontent.com/37161315/234551042-27399b36-e689-4cdb-b700-adabad058053.png">